packages/application-shell/src/utils/get-correlation-id/get-correlation-id.js

```diff
@@ -1,8 +1,13 @@
 import uuid from 'uuid/v4';
 import selectProjectKeyFromUrl from '../select-project-key-from-url';
 
+const VALID_ID_PART_FORMAT = /^[\w-/]+$/;
```

Comment:
would you mind adding a comment explaining what this does for those of us stupid enough not to be regex fluent 😅